docs/integrations/netskope.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Netskope Integration
+category: 62cdf6722c2c1602a4b69643
+hidden: false
+---
+
+# Netskope Integration
+
+## Overview
+
+Phylum can be integrated with [Netskope](https://www.netskope.com) to incorporate Phylum's curated list of malicious open-source package hashes into a Netskope security policy.
+
+## Getting Started
+
+Netskope users who purchase a Phylum integration can follow the instructions below to subscribe to Phylum's curated malicious package hash list distribution. This list will be automatically reflected in your Netskope File Hash List on a regular cadence. There are three separate pieces of information needed to complete the subscription process:
+
+1. A v1 API Token
+2. Your Netskope tenant name
+3. A Netskope File Hash List name
+
+### Obtain a v1 API Key
+
+1. Log into your Netskope admin dashboard and then click on `Settings`.
+2. In `Settings` click on `Tools`.
+3. In `Tools` click on `Rest API v1`.
+4. Grab the Token value from Token section.
+
+### Obtain Your Netskope Tenant Name
+
+1. Your Netskope Tenant name is simply the subdomain of your dashboard URL. For example when you log into your Netskope dashboard, look at the URL. It should be of the form `https://<tenant-name>.goskope.com/ns#/dashboard`. Grab the `<tenant-name>` part of this URL.
```

Comment:
I haven't used Netskope, but do you prefer people to get this name from the URL bar, or from the settings like [this](https://docs.netskope.com/en/netskope-help/integrations-439794/netskope-cloud-exchange/get-started-with-cloud-exchange/configure-netskope-tenants/#:~:text=Go%20to%20Settings%20and%20click,configured%20Netskope%20tenants%20are%20displayed.)?

Reply:
Good question. I'm not sure what the "proper" method is. If we want to write the instruction assuming the user doesn't even have a tenant created yet, then this would definitely be the way to go. I think I'll just do that. I'll rewrite this section to be called "Configure a Tenant" and just point to that link you provided and then say, grab that tenant name. What do you think?

Reply:
I guess they would need to have at least one, and that would be the one in the URL bar. I just don't know how likely it is that they have more than one and need to navigate that. And even if they only have one, should we have them look at the URL bar for the name, or see it in that config page?

Whatever you decide is fine with me!

Reply:
Maybe I'll provide a provisional "if you don't have a tenant configured, follow the directions at the link", otherwise, just grab it from your URL bar.